test/claims-status/components/YourClaimsPageV2.unit.spec.jsx

```diff
@@ -6,31 +6,25 @@ import sinon from 'sinon';
 import { YourClaimsPageV2 } from '../../../src/js/claims-status/containers/YourClaimsPageV2';
```

Comment:
Might make sense to have a sort test... :thinking:
I hate to say this, but just to get this out the door, maybe we ignore it for now if it works with manual testing. We can write the test after everything needed for UAT gets merged.